zaza/openstack/charm_tests/swift/tests.py

```diff
@@ -114,6 +114,17 @@ def test_901_pause_resume(self):
                     'swift-object-replicator',
                     'swift-object-updater',
                     'swift-container-sync']
+
+        focal_victoria = openstack_utils.get_os_release('focal_victoria')
+        if self.current_os_release < focal_victoria:
+            services += ['swift-account-replicator',
+                         'swift-container-replicator',
+                         'swift-object-replicator']
```

Comment:
Shouldn't these be removed from the list above?

Reply:
Good catch :facepalm: